### PR TITLE
Fix institute settings page crash

### PR DESCRIPTION
**Pull request by mhkc**
This PR prevents institute settings page to crash if `collaborators` key is not set for the given institute in the database.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by mhkc** [comment]
The test `tests/utils/test_ensembl_rest_clients.py::test_test_query_biomart_38_xml` failed. Its due to my changes and it could be disregarded.

**Comment by mikaell** [comment]
> The test `tests/utils/test_ensembl_rest_clients.py::test_test_query_biomart_38_xml` failed. Its due to my changes and it could be disregarded.

Crash fixed in 1ee01042d, Bugfix: XML Query Replies not Percent Encoded (#2768).

File: scout/server/blueprints/institutes/templates/overview/utils.html [hunk 1]
```diff
@@ -92,7 +92,7 @@ <h5 class="header">General Institute Settings</h5>
               {{ form.institutes.label(class="control-label",data_toggle="tooltip", data_placement="top", title="Allow case sharing only with preselected institutes.") }}
               <select multiple="multiple" class="selectpicker" name="institutes">
               {% for choice in form.institutes.choices %}
-                <option value="{{choice[0]}}" {% if choice[0] in institute.collaborators %} selected {% endif %}>{{choice[1]}}</option>
+                <option value="{{choice[0]}}" {% if choice[0] in institute.get("collaborators", []) %} selected {% endif %}>{{choice[1]}}</option>
```
Comment (mikaell) [inline scout/server/blueprints/institutes/templates/overview/utils.html:95]: 👍

**Review by mikaell — APPROVED** [review]
Nice and easy fix.

**Comment by mikaell** [comment]
How to test this fix? `Collaborators` is set from `owners` in `parse.case.py:542`. When `owners` is deleted from a config file it will not load, but raise an error.

**Comment by northwestwitch** [comment]
> How to test this fix? `Collaborators` is set from `owners` in `parse.case.py:542`. When `owners` is deleted from a config file it will not load, but raise an error (maybe I'm looking at the wrong place). Further, I cannot find where `institute.collaborators` is set.
> 
> The code looks good. But don't merge until 2nd developer has had the chance to test.

Hi, as suggested by @mikaell if a case has no owner (missing key in the config file) you should get syntax error and the case won't load. Checked also if you provide a non-existent owner in config file: it raises integrity error and the case doesn't load. How did you find yourself in the situation of having a case with no owner/collaborators?

**Comment by northwestwitch** [comment]
To reproduce this error: 
- On master branch, local instance of Scout, create a new institute: `scout --demo load institute -i cust001 -d "Customer 001"`
- Go on the new institute's settings:
```
  File "/Users/chiararasi/Documents/work/GITs/scout/scout/server/templates/layout_bs4.html", line 82, in block 'content'
    {% block content_main %}{% endblock %}
  File "/Users/chiararasi/Documents/work/GITs/scout/scout/server/blueprints/institutes/templates/overview/institute_settings.html", line 33, in block 'content_main'
    {{ institute_settings(form, institute, current_user, loqus_instances) }}
  File "/Users/chiararasi/miniconda3/envs/test_py38/lib/python3.8/site-packages/jinja2/runtime.py", line 828, in _invoke
    rv = self._func(*arguments)
  File "/Users/chiararasi/Documents/work/GITs/scout/scout/server/blueprints/institutes/templates/overview/utils.html", line 95, in template
    <option value="{{choice[0]}}" {% if choice[0] in institute.collaborators %} selected {% endif %}>{{choice[1]}}</option>
jinja2.exceptions.UndefinedError: 'dict object' has no attribute 'collaborators'
```

- Switch to the branch and notice that it works!  

Well done @mhkc!!!

**Pull request merged**